common/src/Microsoft.Azure.IIoT.AspNetCore/src/ResponseTime/Extensions/ResponseTimeMiddlewareEx.cs

```diff
@@ -0,0 +1,22 @@
+// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.Azure.IIoT.AspNetCore.Monitoring {
+    using Microsoft.AspNetCore.Builder;
+
+    /// <summary>
+    ///Response time middleware extensions
+    /// </summary>
+    public static class ResponseTimeMiddlewareEx {
```

Comment:
Couldn't we just use UseHttpMetrics() (https://github.com/prometheus-net/prometheus-net/blob/master/Prometheus.AspNetCore/HttpMetricsMiddlewareExtensions.cs) from Prometheus.aspnetcore package which we are already linking?

Reply:
Didn't know about it. Thanks for sharing.